hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -57,13 +57,20 @@ static bool param_combined_z_axis;
 
 module_param_named(combined_z_axis, param_combined_z_axis, bool, 0644);
 MODULE_PARM_DESC(combined_z_axis,
-		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine");
+		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine.");
 
 static u8 param_trigger_rumble_damping = 0;
+
```

Comment:
This empty line does not match the style of the block just above.

Reply:
The style on this was already inconsistent before my PR, which one is preferred?

Reply:
I think we should stick with how in-kernel drivers do it.

Reply:
I'm not sure what that looks like. The code in `master` doesn't follow a consistent style either.

Reply:
I'll take a peek when I get to work on the code next time and fix this properly.